src/V3Timing.cpp

```diff
@@ -1067,6 +1124,7 @@ class TimingControlVisitor final : public VNVisitor {
             }
             auto* const beginp = VN_AS(stmtp, Begin);
             stmtp = beginp->nextp();
+            if (m_procp->user2() & T_HAS_PROC) beginp->user2(T_HAS_PROC);
```

Comment:
```suggestion
            if (m_procp->user2() & T_HAS_PROC) addFlag(beginp, T_HAS_PROC);
```

Reply:
Like above --  the funcs belong to the graph-building object. And using namespaces for checking bits doesn't seem to make the code clearer.

Reply:
Because of the change above -- this is now also done.